api.go

```diff
@@ -243,13 +243,14 @@ func BootstrapCluster(conf *Config, logs LogStore, stable StableStore,
 // the usual APIs in order to bring the cluster back into a known state.
 func RecoverCluster(conf *Config, fsm FSM, logs LogStore, stable StableStore,
 	snaps SnapshotStore, trans Transport, configuration Configuration) error {
+	var err error
```

Comment:
So, with all these new predeclarations, I wonder if there's a clean way to hoist them up to the top of the function...